helm_deploy/hmpps-github-discovery/templates/_github_teams_discovery_envs.tpl

```diff
@@ -0,0 +1,17 @@
+{{- define "githubTeamsDiscovery.envs" -}}
+{{- if .Values.generic-service.namespace_secrets -}}
```

Comment:
can use a hyphen here, not sure you can reference the values from a subchart like this either.